rethinkdb_base_block.py

```diff
@@ -0,0 +1,94 @@
+import rethinkdb as rdb
+
+from nio.block.base import Block
+from nio.command import command
+from nio.properties import (IntProperty, StringProperty, VersionProperty,
+                            TimeDeltaProperty)
+from nio.util.discovery import not_discoverable
+
+
+@command("Get server info", method="_get_server_info")
+@command("Reconnect to server", method="_reconnect_connection")
+@command("List server databases", method="_get_list_of_databases")
+@command("List database tables", method="_get_list_of_tables")
```

Comment:
Are these a useful feature of this block? I'm not sure that it makes sense to add all these to every ReDB block. Are we using them anywhere? I ask because we have never done this kind of thing before and I don't want to add features that won't be used.

If we do keep them around (I think the reconnect one makes the most sense), then the methods shouldn't have the underscore in front.